packages/origin-247-certificate/src/offchain-certificate/config/configuration.ts

```diff
@@ -0,0 +1,20 @@
+import { Configuration } from './config.interface';
+
+const configuration = (): Configuration => ({
```

Comment:
As this is function please name this `getConfiguration`

Reply:
fixed